var/spack/repos/builtin/packages/py-imgaug/package.py

```diff
@@ -12,6 +12,8 @@ class PyImgaug(PythonPackage):
     homepage = "https://pypi.org/project/imgaug/"
     url = "https://files.pythonhosted.org/packages/4f/96/c55f2985b464ea377238e61026dfb1da54d8edc1534c649e70077e94c642/imgaug-0.2.9.tar.gz"
 
+    # The first version below just serves to trigger a rebuild!
+    version('0.2.9.20210902', url=url, sha256='42b0c4c8cbe197d4f5dbd33960a1140f8a0d9c22c0a8851306ecbbc032092de8', expand=False)
```

Comment:
Isn't this a proper `PythonPackage`? In that case we need to grab the sources, not a full wheel to be installed.
```suggestion
    version('0.2.9.20210902', sha256='42b0c4c8cbe197d4f5dbd33960a1140f8a0d9c22c0a8851306ecbbc032092de8',)
```

Reply:
Yes, you are right. I was slowly coming to this. Fixed in the next commit.